paddle/fluid/inference/tensorrt/engine.h

```diff
@@ -106,21 +106,25 @@ class TensorRTEngine : public EngineBase {
   // Return the output's GPU memory address without copy.
   void* GetOutputInGPU(const std::string& name);
   // Copy data into dst inside the GPU device.
-  void GetOutputInGPU(const std::string& name, void* dst, size_t max_size);
+  void GetOutputInGPU(const std::string& name, void* dst);
   // LOW EFFICENCY! Get output to CPU, this will trigger a memory copy from GPU
   // to CPU.
-  void GetOutputInCPU(const std::string& name, void* dst, size_t max_size);
+  void GetOutputInCPU(const std::string& name, void* dst);
```

Comment:
here, the `max_size` is needed to indicate the size of the `dst` buffer, to assert that the output buffer is smaller than  `max_size`.

Reply:
`dst` can be allocated by user, make sure the buffer is large enough for the `output data`.

Reply:
Done!